source/client/stream_decoder.h

```diff
@@ -81,6 +81,10 @@ class StreamDecoder : public Envoy::Http::StreamDecoder,
   streamResetReasonToResponseFlag(Envoy::Http::StreamResetReason reset_reason);
   void finalizeActiveSpan();
   void setupForTracing(std::string& x_request_id);
+  static const std::string& staticUploadContent() {
```

Comment:
Nit: should be private

Reply:
Done: cb36e1fabb0018fc2a747d5c5e40ef047a3ad962